packages/samples/headless-ssr/package.json

```diff
@@ -6,10 +6,10 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start",
+    "prod": "next start",
     "lint": "next lint",
-    "prod": "npm run build && npm run start",
```

Comment:
yes, and I believe that's the correct behaviour for `prod`. If we run `prod` on Atomic, it will also not work unless we ran `build` first. Additionally, if we use `nx` to run `prod`, it will run `build` first (e.g., `nx run headless-ssr-samples:prod`).